openedx/core/djangoapps/schedules/models.py

```diff
@@ -0,0 +1,18 @@
+from django.db import models
+from django.utils.translation import ugettext_lazy as _
+from django_extensions.db.fields import ModificationDateTimeField
+
+
+class Schedule(models.Model):
```

Comment:
I debated this internally. `started` is essentially created. I opted to not use an `AutoField` because that limits our ability to edit the field in Django admin. I'm open to discussing if we need this option to edit.